### DISCUSSION TITLE
Equivalent to onicecandidate in webrtc demo

### DISCUSSION THREAD

**Discussion by kirankumar-ravuri** [baresip/baresip#1946] [General]
Hi All

What is the equivalent flow for [onicecandidate](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/icecandidate_event#description) in [webrtc demo](https://github.com/baresip/baresip-webrtc)?

BRs
Kiran

**Comment by alfredh** [comment]
We are planning to move baresip-webrtc into baresip, so that the application is
easier to setup and maintain.

We currently dont have "onicecandidate" callback handler, but this is something that could be added.

Please see the API here:
https://github.com/baresip/baresip/blob/main/src/peerconn.c
